pyinjective/client/chain/grpc/chain_grpc_exchange_v2_api.py

```
@@ -575,6 +575,51 @@
 
         return response
 
+    async def fetch_l3_derivative_orderbook(
+        self,
+        market_id: str,
+    ) -> Dict[str, Any]:
+        request = exchange_query_pb.QueryFullDerivativeOrderbookRequest(
```

Pylint (3.3.7)

[error] Module 'pyinjective.proto.injective.exchange.v2.query_pb2' has no 'QueryFullDerivativeOrderbookRequest' member

(E1101)